src/include/mallocMC/creationPolicies/Scatter.hpp

```diff
@@ -1029,7 +1043,7 @@ namespace mallocMC
                             chunksize = alpaka::math::max(
                                 acc,
                                 (uint32) slotSize,
-                                +minChunkSize1); // ensure minimum chunk size
+                                +minChunkSize); // ensure minimum chunk size
```

Comment:
Why the unary plus here?

Reply:
It is a `constexpr` value and this, without the plus the function can not be called.

Reply:
True, as discussed offline that would be an odr-use and our constexpr values normally do not have any storage.